src/services/reverse-geocoder.php

```diff
@@ -5,5 +5,6 @@
 use SearchApi\Models as Models;
 
 interface ReverseGeocoder {
+  function get_url( Models\GeoCoordinate $coords );
```

Comment:
No it really doesn't need the get _url ().. just wanted to be able to test the returned url in the geo-encoder